content/intro-to-storybook/svelte/en/conclusion.md

```diff
@@ -7,19 +7,19 @@ Congratulations! You created your first UI in Storybook. Along the way you learn
 
 [📕 **GitHub repo: chromaui/learnstorybook-code**](https://github.com/chromaui/learnstorybook-code)
 <br/>
-[🌎 **Deployed Storybook**](https://clever-banach-415c03.netlify.com/)
+[🌎 **Deployed Storybook**](https://clever-banach-415c03.netlify.app/)
 
 Storybook is a powerful tool for React, React Native, Vue, and Angular. It has a thriving developer community and a wealth of addons. This introduction scratches the surface of what’s possible. We’re confident that once you adopt Storybook, you’ll be impressed by how productive it is to build durable UIs.
 
 ## Learn more
 
 Want to dive deeper? Here are helpful resources.
 
-- [**Official Storybook documentation**](https://storybook.js.org/basics/introduction/) has API documentation, community links, and the addon gallery.
+- [**Official Storybook documentation**](https://storybook.js.org/docs/react/get-started/introduction) has API documentation, community links, and the addon gallery.
```

Comment:
@gregrickaby when can please update from "/docs/react/get-started/introduction" to "/docs/svelte/get-started/introduction" in the "Official Storybook documentation" link.